app/src/main/java/com/example/soundcloud/data/source/remote/GetDataFromAPI.java

```diff
@@ -0,0 +1,67 @@
+package com.example.soundcloud.data.source.remote;
+
+import com.example.soundcloud.data.model.DataHelper.SoundCloud;
+import com.example.soundcloud.data.model.Song;
+import com.example.soundcloud.data.model.SongJSonKey;
+import com.example.soundcloud.data.model.SongType;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetDataFromAPI {
```

Comment:
Tên class là 1 danh từ
```suggestion
public class SongLoaderUtils {
```